### PR TITLE
Basic Socket communication from android to server

### PR DESCRIPTION
**Pull request by kbumsik**
I firstly added socket communication between phone and server. I will start building RawDB server then.

## How to use

On server:
```
$ python3 backend/comm.py
```
and then run the app and try sending. If you want to test server without the app, use `$ python3 backend/scripts/connection_test.py`

### CODE REVIEW

File: ActivityRecognition/.idea/gradle.xml [hunk 1]
```diff
@@ -3,9 +3,9 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="distributionType" value="DEFAULT_WRAPPED" />
+        <option name="distributionType" value="LOCAL" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleHome" value="$APPLICATION_HOME_DIR$/gradle/gradle-2.10" />
+        <option name="gradleHome" value="C:\Office\Android\Android Studio\gradle\gradle-2.14.1" />
```
Comment (kbumsik) [inline ActivityRecognition/.idea/gradle.xml]: What the hack? I am gonna fix this. lol

**Comment by cjackie** [comment]
@kbumsik 
Let me test it before merging. 

About the python script. Python 2 and 3 are not much different, but the thing is Python 3 is not backward compatible with Python 2. So, could you use Python 2 whenever possible. I will your script with Python 2 anyway.

**Comment by kbumsik** [comment]
oh is your project written in python2?
Otherwise requested I use 3 cuz 2 is looking at [end of life](http://legacy.python.org/dev/peps/pep-0373/). There will be no update on 2 but only bugfixs. And most of Linux distros already dropping python 2.7 (e.g. [Ubuntu as of 16.04 LTS](https://wiki.ubuntu.com/Python/FoundationsXPythonVersions)) So is there any package only support 2? Anyway I will rewrite it for 2 if you want.

**Comment by cjackie** [comment]
@kbumsik 
I see, let's stick with Python 2 for now. I feel like most of people are still using Python 2.

**Comment by kbumsik** [comment]
OK I changed it. You can use this with Python 2 interpreter :)

**Pull request merged**